google/cloud/speech/cloud_speech.yaml

```diff
@@ -12,18 +12,8 @@ documentation:
 
 apis:
 - name: google.cloud.speech.v1beta1.Speech
-# TODO get LRO API call support added for Speech
-#- name: google.longrunning.Operations
+- name: google.longrunning.Operations
```

Comment:
AFAIK the main difference is that when you use a service as a mixin, you get to redefine the HTTP bindings. For GAPIC, this means that we can generate PathTemplate helpers for the IAM methods. This makes sense because IAM operates over resources. In contrast, LRO operates over objects so it doesn't need helper methods into the resource model of the API.